packages/components/src/radio-group/index.ts

```diff
@@ -5,22 +5,44 @@ import {
   Selector,
   StateConfigMap,
   Store,
+  UpdateStateAction,
 } from '@devextreme/core';
-import { UpdateStateAction } from '@devextreme/core/src/store';
+import {
+  DEFAULT_ROOT_CONTAINER_PROPS,
+  ROOT_CONTAINER_PROP_BUILDERS,
+  RootContainerDomOptions,
+  RootContainerProps,
+} from '../root-container';
+
+// === base component ===
+export type RadioGroupContainerDomOptions = RootContainerDomOptions['accessKey']
```

Comment:
Suggestion:
More type safer variant (changing in chain directly affected to the type)
```
export type RadioGroupContainerDomOptions = typeof RADIO_GROUP_CONTAINER_PROPS_BUILDER
```
Remove export because we have already aliase this type to ReadonlyProps

Reply:
Fixed